client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationNumberColumn/InvestigationStatusNumberStyles.ts

```diff
@@ -0,0 +1,16 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    columnWrapper: {
+        display: 'flex'
+    },
+    transferedIcon: {
+        fontSize: '20px',
+        margin: '0.2vh 0.5vw',
+        cursor: 'pointer',
+        color: theme.palette.primary.main,
```

Comment:
shouldnt you just use the color prop on the elemnt itself?

Reply:
doesn't work on element